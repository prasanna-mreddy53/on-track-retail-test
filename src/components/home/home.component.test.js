import React from "react";
import Home from "./home.component";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

let match = {
  params: {
    page: 1,
  },
};
const wrapper = shallow(<Home match={match} />);

describe("Home component tests", () => {
  it("should match snapshot", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
