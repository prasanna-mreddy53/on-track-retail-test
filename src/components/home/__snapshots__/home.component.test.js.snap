// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component tests should match snapshot 1`] = `
<div
  className="App"
>
  <Header
    searchBooks={[Function]}
  />
  <div>
    Loading...
  </div>
  <content>
    <Row
      className="m-0"
      noGutters={false}
    />
    <PaginationComponent
      activePage={1}
      currentPage={1}
      getPageNumber={[Function]}
      itemsPerPage={20}
      nextPage={[Function]}
      prevPage={[Function]}
      totalBooks={0}
    />
  </content>
</div>
`;
